src/main/resources/data.sql

```diff
@@ -648,3 +648,20 @@ insert into address (id, user_id, line1, line2, city, state, zip) values
 ('42f33d30-f3f8-4743-a94e-4db11fdb747d', '008a4215-0b1d-445e-b655-a964039cbb5a', '412 Maple St', null, 'Dowagiac', 'Michigan', '49047'),
 ('579872ec-46f8-46b5-b809-d0724d965f0e', '00963d9b-f884-485e-9455-fcf30c6ac379', '237 Mountain Ter', 'Apt 10', 'Odenville', 'Alabama', '35120'),
 ('95a983d0-ba0e-4f30-afb6-667d4724b253', '00963d9b-f884-485e-9455-fcf30c6ac379', '107 Annettes Ct', null, 'Aydlett', 'North Carolina', '27916');
+
+INSERT INTO payment (id, user_id, card_number, expiry_month, expiry_year, updated) VALUES
+('0001', '008a4215-0b1d-445e-b655-a964039cbb5a', '4532016747591234', 1, 2024, NOW()),
+('0002', '008a4215-0b1d-445e-b655-a964039cbb5a', '6011514433546201', 2, 2025, NOW()),
+('0003', '008a4215-0b1d-445e-b655-a964039cbb5a', '6771549495588452', 3, 2023, NOW()),
+('0004', '008a4215-0b1d-445e-b655-a964039cbb5a', '4716350812345678', 4, 2024, NOW()),
+('0005', '008a4215-0b1d-445e-b655-a964039cbb5a', '6011123456789123', 5, 2025, NOW()),
+('0006', '008a4215-0b1d-445e-b655-a964039cbb5a', '5252803995123456', 6, 2026, NOW()),
+('0007', '008a4215-0b1d-445e-b655-a964039cbb5a', '6011012345678912', 7, 2027, NOW()),
+('0009', '008a4215-0b1d-445e-b655-a964039cbb5a', '3584587763245678', 9, 2024, NOW()),
+('0010', '008a4215-0b1d-445e-b655-a964039cbb5a', '6011000990139424', 10, 2025, NOW()),
+('0011', '008a4215-0b1d-445e-b655-a964039cbb5a', '4532016747591287', 11, 2026, NOW()),
+('0012', '008a4215-0b1d-445e-b655-a964039cbb5a', '6011514433546207', 12, 2027, NOW()),
+('0013', '008a4215-0b1d-445e-b655-a964039cbb5a', '6011313207071335', 1, 2023, NOW()),
+('0014', '008a4215-0b1d-445e-b655-a964039cbb5a', '4024007162397651', 2, 2024, NOW()),
+('0017', '008a4215-0b1d-445e-b655-a964039cbb5a', '6011514433546211', 6, 2027, NOW());
```

Comment:
- Any reason why you choose to use this `id` pattern? It seems problematic when you try to add the 10,000 record. 
- You should have added some different `user_id` values as well. Seems like this 1 user has way too many credit cards. 😏 
- `updated` has a `default` rule when excluded so you can achieve the same result by dropping the `updated` column altogether. note: `updated timestamp not null default current_timestamp()`.

Reply:
All valid, again I don't have the strongest SQL background as the majority of my experience is in service aggregation for a BFF architecture.